components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java

```diff
@@ -222,6 +226,14 @@ public static void validateDomain(String domainName) throws Exception {
                 throw new TenantManagementClientException(ERROR_CODE_EMPTY_EXTENSION);
             }
         }
+        String regex = CommonUtil.getTenantDomainRegexPattern();
```

Comment:
Put a comemnt saying, the regex validation will be skipped if it is not configured.